sdk/src/feature_set.rs

```diff
@@ -311,6 +311,10 @@ pub mod vote_withdraw_authority_may_change_authorized_voter {
     solana_sdk::declare_id!("AVZS3ZsN4gi6Rkx2QUibYuSJG3S6QHib7xCYhG6vGJxU");
 }
 
+pub mod spl_associated_token_account_v1_0_4_release {
```

Comment:
feel free to ignore, the "_release" feels superfluous and unnecessarily verbose to me
```suggestion
pub mod spl_associated_token_account_v1_0_4 {
```